api/src/error_email/error_email_template.html

```diff
@@ -0,0 +1,62 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8" />
+    <title>CV Manager API Error Email</title>
+    <meta name="generator" content="Google Web Designer 15.2.1.0306" />
+    <style id="gwd-text-style">
+      p {
+        margin: 0;
+      }
+      h1 {
+        margin: 0;
+      }
+      h2 {
+        margin: 0;
+      }
+      h3 {
+        margin: 0;
+      }
+    </style>
+    <style>
+      html,
+      body {
+        width: 100%;
+        height: 100%;
+        margin: 0;
+      }
+      body {
+        background-color: transparent;
+        transform: perspective(1400px) matrix3d(1, 0, 0, 0, 0, 1, 0, 0, 0, 0, 1, 0, 0, 0, 0, 1);
+        transform-style: preserve-3d;
+      }
+      body * {
+        transform-style: preserve-3d;
+      }
+    </style>
+  </head>
+  <body>
+    <p>
+      You are receiving this email because your user is marked to receive error emails on the CV Manager Admin page.
+    </p>
+    <br />
+    <p>This error originated in the ##_ENVIRONMENT_## environment CV Manager API</p>
+    <br />
+    <p>Error Message: ##_ERROR_MESSAGE_##</p>
+    <br />
+    <p>Error occurred at: ##_ERROR_TIME_##</p>
+    <br />
+    <p>View this error in Cloud Run logs: <a href="##_CLOUD_RUN_LOGS_LINK_##">rsu-manager-cloud-run-api logs</a></p>
+    <br />
+    <p>--</p>
+    <p>
+      If you no longer wish to receive these emails, please uncheck the "receive error emails" checkbox on the CV
```

Comment:
Doesn't need to be in this PR, but I think future-state here will be an "unsubscribe" link that directly takes the user out of the email list.

Reply:
Yeah, agree. Maybe we could create an unauthenticated endpoint to the API that allows user to unsubscribe?

Reply:
Added an unauthenticated unsubscribe endpoint